ext/pdo_pgsql/tests/bug62498-32bit.phpt

```diff
@@ -207,7 +213,7 @@ array(9) {
     ["pgsql:table_oid"]=>
     int(%d)
     ["table"]=>
-    string(13) "bugtest_62498"
+    string(%s) "test62498_32"
```

Comment:
String seems to have known length?

Reply:
Yes, I changed it to `%s` because we are not interested in the length and I don't want to change the length again if the table name changes for any reason.